src/common/requests.ts

```diff
@@ -437,6 +438,25 @@ function makeRequest(
         /*initializer*/ ts.factory.createIdentifier("payload")
       )
     );
+    if ("content" in item.requestBody) {
+      axiosConfigFields.push(
+        ts.factory.createPropertyAssignment(
+          /*name*/ ts.factory.createIdentifier("headers"),
+          /*initializer*/ ts.factory.createObjectLiteralExpression(
+            /*properties*/ [
+              ts.factory.createPropertyAssignment(
+                /*name*/ ts.factory.createStringLiteral(`Content-Type`),
+                /*initializer*/ ts.factory.createStringLiteral(
+                  Object.keys(item.requestBody.content).at(0) ??
+                    "application/json"
```

Comment:
If there is no value inside content[0] then I suggest we don't default to json, we should just not push to the `axiosConfigFields` array if that is the case